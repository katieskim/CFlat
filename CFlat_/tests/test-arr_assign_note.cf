int main()
{
  note[] note_arr;
  note n;

	n = ( /C-/ /4/ /q/ );
  note_arr = (note[]) [n,n,n,n];  
  print(note_arr[1]);               
  (: should print out the array literal, the output file for this test is probably wrong :) 
  return 0;
}
