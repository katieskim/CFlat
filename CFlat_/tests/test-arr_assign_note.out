/C-/ /4/ /q/